components/agenda/src/Agenda.svelte

```diff
@@ -589,11 +594,12 @@
       (canvas.height - mouseYPosition) / canvas.height,
     ];
 
+    const max = 1000;
```

Comment:
can we have this be a top level const, so that it can be set as the default value of the `end_minute` prop? Probably also should be renamed to something like `MAX_MINUTES_IN_VIEW`